apps/src/metrics/MetricsReporter.ts

```diff
@@ -163,13 +173,16 @@ class MetricsReporter {
         name: 'Browser',
         value: getBrowserName(),
       },
-      {
-        name: 'BrowserVersion',
-        value: getBrowserName(true),
-      },
     ];
   }
 
+  private getBrowserVersionDimension(): MetricDimension {
+    return {
+      name: 'BrowserVersion',
+      value: getBrowserName(true),
```

Comment:
Nit: could rename `getBrowserName` to `getBrowerInfo` since version can be included.

Reply:
Hmm true, though I think I kept it this way since both do technically contain the browser name (i.e. "Chrome" vs "Chrome 129")